### PR TITLE
feat: record subnets/participants/instances per task

### PR DESCRIPTION
**Pull request by bajtos**
Help us understand how many data points we have for each task (what is the effective committee size). Are the committees always large enough to give us enough confidence in finding an honest majority?

Also, record how many measurements (retrieval checks) we provide for each task to understand how much load we put on SPs.

InfluxDB point name: `committees`
Field prefixes:
- `subnets_`
- `participants_`
- `instances_`
- `measurements_` (all measurements, independent of fraud assessment)

Links:
- https://github.com/filecoin-station/spark-api/pull/171
- https://github.com/filecoin-station/spark/issues/45
- https://github.com/filecoin-station/roadmap/issues/52

### CODE REVIEW

File: lib/retrieval-stats.js [hunk 1]
```diff
@@ -143,3 +143,45 @@ const countUniqueTasks = (measurements) => {
...
+        subnets: new Set(),
+        participants: new Set(),
+        instances: new Set()
+      }
+      tasks.set(key, data)
+    }
+    data.subnets.add(m.inet_group)
+    data.participants.add(m.participantAddress)
+    // We don't have Station instance identifier in the measurement.
+    // The pair (inet_group, participant_address) is a good approximation.
```
Comment (juliangruber) [inline lib/retrieval-stats.js:168]: nice!

File: lib/retrieval-stats.js [hunk 2]
```diff
@@ -92,15 +92,15 @@ export const buildRetrievalStats = (measurements, telemetryPoint) => {
  * @param {string} fieldNamePrefix
  * @param {number[]} values
  */
-const addHistogramToPoint = (point, fieldNamePrefix, values) => {
+const addHistogramToPoint = (point, values, fieldNamePrefix = '') => {
```
Comment (juliangruber) [inline lib/retrieval-stats.js:95]: why was this changed? Iiuc we never pass an empty prefix

**Review by juliangruber — CHANGES_REQUESTED** [review]

File: lib/retrieval-stats.js (hunk 2 above)
Comment (bajtos) [reply lib/retrieval-stats.js:95]: It did pass an empty prefix in the first iteration; see [658cb1c](https://github.com/filecoin-station/spark-evaluate/pull/97/commits/658cb1ced84718babfe090eb0a94c097991140e3).

I think this can be useful again in the future, so I kept it.

**Review by juliangruber — APPROVED** [review]
